.azure-pipelines/templates/osx/pack.signed/step5-dist.yml

```diff
@@ -0,0 +1,36 @@
+steps:
+  - task: DownloadPipelineArtifact@1
+    displayName: Download signed installer
+    inputs:
+      buildType: 'current'
+      artifactName: 'tmp.macinstaller_signed'
+      downloadPath: '$(Build.StagingDirectory)\pkg'
+
+  - task: DownloadPipelineArtifact@1
+    displayName: Download signed payload
+    inputs:
+      buildType: 'current'
+      artifactName: 'tmp.macpayload_signed'
+      downloadPath: '$(Build.StagingDirectory)\payload'
+
+  - task: DownloadPipelineArtifact@1
+    displayName: Download symbols
+    inputs:
+      buildType: 'current'
+      artifactName: 'tmp.macsymbols'
+      downloadPath: '$(Build.StagingDirectory)\symbols'
+
+  - script: src/osx/SignFiles.Mac/notarize-pkg.sh -id "$(AppleId)" -p "$(AppleIdPassword)" -pkg '$(Build.StagingDirectory)\pkg\*.pkg'
```

Comment:
Won't we need to also notarize the platform-native auth helper?

Reply:
Notarization happens recursively inside of PKG package files and DMG disk images.
You can notarize the following things:
1. Bundles (.app, .kext, etc)
2. Installer package files
3. Disk images
4. Mach-o binaries (like the MSAuth helper UNIX executable)

You can staple the following things:
1. Bundles
2. Installer package files
3. Disk images

Flat binaries cannot be stapled, but are still notarized either by manual submission, or submission as part of a 'container'.

Reply:
Ah, thanks for clarifying!